frontends/web/src/routes/account/summary/accountssummary.tsx

```diff
@@ -103,6 +103,9 @@ class AccountsSummary extends Component<Props, State> {
 
   private getAccountSummary = () => {
     return accountApi.getSummary().then(data => {
+      for (let c in data.ratesTimestamps) {
+        console.log(c + ': ' + new Date(data.ratesTimestamps[c as accountApi.CoinCode]).toLocaleString());
```

Comment:
This is just to show how to handle the data, will be probably removed when the frontend starts using the timestamps to show meaningful messages.